test/parallel/test-child-process-validate-stdio.js

```diff
@@ -30,6 +30,21 @@ common.expectsError(() => _validateStdio(stdio2, true),
                     { code: 'ERR_IPC_SYNC_FORK', type: Error }
 );
 
+// should throw if stdio is not a valid input
+{
+  const stdio3 = ['foo'];
+  common.expectsError(() => _validateStdio(stdio3, false),
+                      { code: 'ERR_INVALID_SYNC_FORK_INPUT', type: TypeError }
+  );
+}
+
+// should throw if stdio is not a valid option
+{
+  const stdio4 = [{ foo: 'bar' }];
+  common.expectsError(() => _validateStdio(stdio4, false),
+                      { code: 'ERR_INVALID_OPT_VALUE', type: TypeError }
```

Comment:
I think we can reuse `expectedError` above here.
https://github.com/nodejs/node/blob/7e0ddf66b9e09a27e0761b487342027ed6284027/test/parallel/test-child-process-validate-stdio.js#L8-L9

Something like:
```js
const expectedError = 
   common.expectsError({ code: 'ERR_INVALID_OPT_VALUE', type: TypeError }, 3);

...

assert.throw(() => _validateStdio([{ foo: 'bar' }], false), expectedError)
```

Reply:
Sr yes sr! Thank you!